### ISSUE TITLE
Multi domain certifications should not be shortened to GRC.

### ISSUE THREAD

**Issue by lucasblanca** [PaulJerimy/SecCertRoadmapHTML#153]
"Shortened Security+, SSCP, GSEC, Programming Languages, CASP+, CISSP, CISSP Concentrations, and GSE to cover 4 spaces in only GRC to make room for more certifications"

These are not GRC certifications, they are multi-domain certifications. For GSE you actually need to pass different certifications in other domains.
The 2023 table made sense for all those certifications.
This makes the table objectively wrong regarding those certifications. The justification for the change was very honest, but shows clearly that it is not related to the domains covered by the certifications or other technical aspects.

**Comment by smurfhunter** [comment]
Duplicate of #135

**Comment by sinecurelife** [comment]
> "Shortened Security+, SSCP, GSEC, Programming Languages, CASP+, CISSP, CISSP Concentrations, and GSE to cover 4 spaces in only GRC to make room for more certifications"

I agree and am looking for the best way to display that. The previous way was also inconsistent. I think I have a good solution in the works, but in the meantime I'll revert it back to the "2023" display where the core certs touch all their domains.
